drivers/soundwire/bus_type.c

```diff
@@ -105,6 +105,11 @@ static int sdw_drv_probe(struct device *dev)
 	if (slave->ops && slave->ops->read_prop)
 		slave->ops->read_prop(slave);
 
+	/* init the sysfs as we have properties now */
+	ret = sdw_slave_sysfs_init(slave);
+	if (ret < 0)
+		dev_warn(dev, "Slave sysfs init failed:%d\n", ret);
```

Comment:
@plbossart Hmm, shouldn' t sysfs init fail with error (versus debugfs init where init errors are supposed to be ignored)?

Reply:
I am not sure, it wouldn't prevent the driver from operating, this sysfs init only exposes properties.
I don't mind changing this, it's just that I don't know the expectation.

Reply:
@plbossart I was also just grepping through other drivers and results are pretty mixed. OTOH, in some drivers the sysfs interface is integral part of the userspace interface, so failing upon error makes more sense. In this case, driver continues to work, so maybe this is ok like this (there are prior examples of this as well).

Reply:
no change here